functions/Remove-DbaLinkedServer.ps1

```diff
@@ -107,7 +116,11 @@ function Remove-DbaLinkedServer {
 
             if ($Pscmdlet.ShouldProcess($lsToDrop.Parent.Name, "Removing the linked server $($lsToDrop.Name) on $($lsToDrop.Parent.Name)")) {
                 try {
-                    $lsToDrop.Drop()
+                    if (Test-Bound Force) {
+                        $lsToDrop.Drop($true)
+                    } else {
+                        $lsToDrop.Drop()
+                    }
```

Comment:
Could you not just do this as `$lsToDrop.Drop([boolean]$Force)`? The method can accept a boolean value so if the user does not provide `-Force` that should equate to doing `Drop($false)`.

Though I do not have an lab to test this on.

Reply:
sounds good, updated the code